codex-ui/src/vue/components/alert/AlertContainer.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div
+    :class="[
+      $style.alert__container,
+      $style['alert__container--' + props.position]
+    ]"
+  >
+    <AlertTransition>
+      <Alert
+        v-for="(alert, index) in alerts"
+        :key="index"
+        v-bind="alert"
+      />
+    </AlertTransition>
+  </div>
+</template>
+
+<script setup lang="ts">
+import Alert from './Alert.vue';
+import AlertTransition from './AlertTransition.vue';
+import { useAlert } from './useAlert';
+import type { AlertOptions } from './Alert.types';
+import { genId } from './constant';
+
+const props = withDefaults(defineProps<AlertOptions>(), {
+  id: genId(),
+  position: 'bottom-center',
+  content: '',
+  icon: '',
+  type: undefined,
+  timeout: 5000,
```

Comment:
it's better to leave the only place with default values. I think this properties are note related to Alerts Container

Reply:
probably,`alerts` returned by `useAlert()` should include them

Reply:
its used in Alert container position

Reply:
> it's better to leave the only place with default values. I think this properties are note related to Alerts Container

its used in Alert container position

Reply:
> probably,`alerts` returned by `useAlert()` should include them

yes I added a defaultAlertOpt on `useAlert()
`